source/GameAction.h

```diff
@@ -80,6 +81,9 @@ class GameAction {
 	std::vector<std::pair<const Ship *, std::string>> giftShips;
 	std::map<const Outfit *, int> giftOutfits;
 
+	std::vector<DataNode> hardpoints;
+	std::vector<Outfit> attributes;
```

Comment:
This doesn't work, because the ship will save the outfit in its outfit list but these additional outfits have no name.

Reply:
I will work out another way

Reply:
another way created